snippets/product-images-showcase.liquid

```diff
@@ -1,15 +1,27 @@
+{{ 'splide.min.css' | asset_url | stylesheet_tag }}
+{{ 'splide.min.js' | asset_url | script_tag }}
+
 <section class='product-images {% if hidden %} hidden{% endif %}'>
   <div class='product-images-container'>
-    <div class='thumbnail-images'>
-      {% for image in images %}
-        <div class="image-container" onmouseenter='previewProductImage(this)'>
-          <img
-            loading='lazy'
-            src='{{ image }}'
-            alt='{{ product.name }}'>
-        </div>
-      {% endfor %}
+    {% if images.size > 1 %}
+    <div class='thumbnail-images splide yc-slider__{{ view }}' id='yc_slider_{{id}}'>
+      <div class="splide__track">
+        <ul class="splide__list">
+          {% for image in images %}
+            <li class="image-container splide__slide" onmouseenter='previewProductImage(this)'>
+                <img
+                  loading='lazy'
+                  class='thumbnail-image'
+                  src='{{ image | img_url: 'thumb' }}'
+                  data-image='{{ image | img_url: '1024x1024' }}'
```

Comment:
could you explain to me what data-image attribute for ?

Reply:
okay and for object cover or contain I will leave the product team decide what's better no need for discuss on it in this PR